app/react-native/package.json

```diff
@@ -56,7 +56,7 @@
     "json-loader": "^0.5.4",
     "json5": "^0.5.1",
     "postcss-loader": "^2.0.5",
-    "react-native-compat": "0.0.2",
+    "react-native-compat": "0.0.4",
```

Comment:
Think it really depends on how you plan to maintain it. Automatic updates should not be breaking changes. Typically `^` minor version updates are feature improvements, `~` are bug fixes. If you follow the same convention, `^` should be fine.